packages/ndla-ui/src/ComponentCurser/index.tsx

```diff
@@ -0,0 +1,2 @@
+import ComponentCurser from './ComponentCurser';
```

Comment:
Skulle denne egentlig hett ComponentCursor? For det er vel ikkje banning i bildet her? :)

Reply:
Ha, godt poeng. Ble muligens noen gloser under utvikling. Jeg skal rename den @gunnarvelle 
Caser kom fra nytt design, men bare gi meg en lyd her så oppdaterer jeg den og.